.github/workflows/ci.yml

```diff
@@ -137,7 +137,7 @@ jobs:
     strategy:
       fail-fast: true
       matrix:
-        python-version: ["3.6", "3.7", "3.8", "3.9"]
+        python-version: ["3.6", "3.7", "3.8", "3.9", "3.10"]
```

Comment:
3.10 is interesting, trying to find the article but read somewhere that 3.10 and `black` don't interoperate accurately right now.

Reply:
tests passed :)